### PR TITLE
Added the bwa mem as a callout to the automation script (conflict free)

### PR DESCRIPTION
**Pull request by fpsom**
Updated the [original PR](https://github.com/datacarpentry/wrangling-genomics/pull/6) so that it does not have any conflicts with the current version.

If everything is ok, we can merge and close both PRs.

### CODE REVIEW

**Comment by crazyhottommy** [comment]
so we decided to go with `bwa mem` ? I thought you wanted to keep the current `bwa aln`. I think `bwa mem` is OK. If no other questions, we can merge this.

**Comment by fpsom** [comment]
@crazyhottommy I just added `bwa mem` as a callout at the end (plus as an exercise to put together an alternative script), so the original `bwa aln` should be as is (unless I did something wrong during the merge).

**Comment by fpsom** [comment]
Just realized - I didn't change the title of the PR, so that may be the source of confusion...  :) I updated it, so it should better reflect the actual change. Sorry about that!

File: _episodes/02-automating_a_workflow.md [hunk 1]
```diff
@@ -248,14 +251,43 @@ Filter the SNPs for the final output
 
     $ bcftools view $variants | /usr/share/samtools/vcfutils.pl varFilter - > $final_variants
 
-> ## Exercise
-> Run the script: dcuser/dc_sample_data/variant_calling/run_variant_calling.sh
+> # Using `bwa mem` instead of `bwa aln`
+> BWA is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome, and it's freely available [here](http://bio-bwa.sourceforge.net). It consists of three algorithms: BWA-backtrack, BWA-SW and BWA-MEM, each being invoked with different sub-commands: `aln/samse/sampe` for BWA-backtrack, `bwasw` for BWA-SW and `mem` for the BWA-MEM algorithm. In this lesson we have been using the `aln` for performing the alignment, but the same process can be performed with `bwa mem` which doesn't require the creation of the index files. The process is modified starting from `mkdir` step, and omitting all directories relevant to the `.sai` index files, i.e.:
```
Comment (lexnederbragt) [inline _episodes/02-automating_a_workflow.md]: `aln + samse  + sampe` for BWA-backtrack, 

Also, please explain that read length is most important reason to choose mem over aln (I forgot the cutoff, was i 75 bp?)

**Comment by fpsom** [comment]
Thanks @lexnederbragt. I just addressed both comments and it is definitely more informative now.

**Comment by crazyhottommy** [comment]
thanks @fpsom !

**Pull request merged**